packages/utils.functionrewrite/spec/functionRewriteBehavior.ts

```diff
@@ -14,7 +14,8 @@ describe('Function Rewrite Provider', function () {
       'x: function () { return {} }': 'x: () => { }',
       'x: function () { return "abc" }': 'x: () => "abc"',
       'stringLiteral: "hello", numberLiteral: 123, boolLiteralTrue: true, boolLiteralFalse: false, objectLiteral: {}, functionLiteral: function() { }, nullLiteral: null, undefinedLiteral: undefined': 'stringLiteral: "hello", numberLiteral: 123, boolLiteralTrue: true, boolLiteralFalse: false, objectLiteral: {}, functionLiteral: () => undefined, nullLiteral: null, undefinedLiteral: undefined',
-      'function (v) { return v.name + " (" + v.job + ")"; }': '(v) => v.name + " (" + v.job + ")";'
+      'function (v) { return v.name + " (" + v.job + ")"; }': '(v) => v.name + " (" + v.job + ")"',
+      'function () { foo(); }': '() => foo() && undefined'
```

Comment:
A safer way to return `undefined` is `void 0`, but no modern browser lets you overwrite `undefined` now so I think this is fine.

See e.g.: https://stackoverflow.com/questions/5716976